card/mod/05_standard/set/all/links.rb

```diff
@@ -32,16 +30,16 @@ def smart_link link_text, target, html_args={}
     end
   end
 
-
   # link to a specific url or path
   def web_link href, opts={}
     text = opts.delete(:text) || href
-    new_class = case href
-      when /^https?:/                      ; 'external-link'
-      when /^mailto:/                      ; 'email-link'
-      when /^([a-zA-Z][\-+\.a-zA-Z\d]*):/  ; $1 + '-link'
+    new_class =
+      case href
+      when /^https?\:/                     then 'external-link'
+      when /^mailto\:/                     then 'email-link'
+      when /^([a-zA-Z][\-+\.a-zA-Z\d]*):/  then $1 + '-link'
       when /^\//
-        href = internal_url href[1..-1]    ; 'internal-link'
+        (href = internal_url href[1..-1]) ;     'internal-link'
```

Comment:
Do not use semicolons to terminate expressions.<br>Space found before semicolon.